2_FirstComponent.md

```diff
@@ -154,16 +108,30 @@ The `scope` object is the can.Component's view model. The view model is an abstr
 ![](images/2_first_component/ComponentScopeTemplateLink.png)
 
 #####can.Map &amp; can.List
-The scope is a special type of object, called a "can.Map". can.Map objects are observable. Observable objects provide a way for you to listen for and keep track of changes to them. What this means, in this instance, is that if you make a change to your scope, those changes will be reflected automatically in your view. If you've cross-bound the values between your scope and your view, changes to your view will also be reflected in your scope. We'll see how this works in the next chapter.
+The scope is a special type of object, called a "can.Map". can.Map objects are observable. Observable objects emit events whenever their properties change. can.view (and subsequently can.Component) subscribe to these events and update the DOM as needed. If you've cross-bound the values between your scope and your view, changes to your view will also be reflected in your scope. We'll see how this works in the next chapter.
+
```

Comment:
Or maybe it should be moved into it's own section? I think going into `can.Map` and `can.List` is way more important than `can.Construct`.